src/components/Home/Home.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Carousel from 'react-multi-carousel';
+import { fetchMotorcycles } from '../../redux/motorcycles/motorcycles';
+import Motor from './Motor';
+import responsive from '../../config/responsive';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMotorcycles());
+  }, []);
+  const motors = useSelector((state) => state.motorcycle);
```

Comment:
The state does not have the name `motorcycle` but `motorcycles`. Please do the necessary correction.

Reply:
Hello @Achilles-Dev Thank your for your comment but its `motorcycle` in 
```
const reducer = combineReducers({
  motorcycle: motorcylesReducer,
  auth: authReducer,
  reservations: reservationsReducer,
});
```